practice/src/css/components/button.css

```diff
@@ -0,0 +1,29 @@
+.btn-primary{
+  border: none;
+  border-radius: 12px;
+  text-decoration: none;
+  align-items: center;
+  background: var(--fouth-background-color);
+  color: var(--primary-text-color);
+  cursor: pointer;
+  font-family: var(--primary-font);
+  font-size: var(--font-size-medium);
+}
+
+.btn-primary:hover {
+  background: var(--primary-background-color);
+  color: var(--secondary-text-color);
+}
+
+.btn-secondary {
+  border-radius: 100px;
+  border: 1.5px solid var(--primary-border-color);
+  background: var(--primary-background-color);
+  color: var(--tertiary-text-color);
+  cursor: pointer;
+  font-family: var(--primary-font);
+  text-decoration: none;
+  align-items: center;
```

Comment:
btn-primary and btn-secondary have some common styles. You can define a class `btn` to style the common aspects of the buttons to avoid duplicate code

.btn {
   text-decoration: none;
   align-items: center;
   font-family: var(--primary-font);
   .....
}